### PR TITLE
feat: support ip address match

### PR DESCRIPTION
**Pull request by tzssangglass**

### CODE REVIEW

**Review by kingluo — APPROVED** [review]

File: lib/resty/expr/v1.lua [hunk 1]
```diff
@@ -136,6 +143,7 @@ local compare_funcs = {
     end,
     ["in"] = in_array,
     ["has"] = has_element,
+    ["ipmatcher"] = ip_matcher,
```
Comment (spacewander) [inline lib/resty/expr/v1.lua]: It would be better to avoid using nouns as an operator. `Client address ipmatcher 127.0.0.1` is not a good sentence.

Comment (tzssangglass) [reply lib/resty/expr/v1.lua]: what about `match`?

Comment (spacewander) [reply lib/resty/expr/v1.lua]: > what about `match`?

`ipmatch` would be better?

Comment (tzssangglass) [reply lib/resty/expr/v1.lua]: SGTM

**Review by spacewander — APPROVED** [review]

**Pull request merged**